content/en/blog/2025/sail-operator-ga/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: "Sail operator 1.0.0 has been released: manage Istio with an operator"
+description: Sail Operator 1.0.0 released, a project part of the istio-ecosystem organization. Let's dive in into the basics of the Sail Operator and let's run an example to show how easy it is to manage Istio with it.
```

Comment:
see https://deploy-preview-16332--preliminary-istio.netlify.app/latest/blog/2025/sail-operator-ga/ - this is too long

```suggestion
description: Dive in into the basics of the Sail Operator and check out an example to see how easy it is to use it to manage Istio.
```